### PR TITLE
Features/admin routing 140

### PR DESCRIPTION
**Pull request by iandusenbury**
Admin page still needs functionality for regular admins to add an admin, wanted to push this out because there are various other fixes (most notably to the api.js file that allows post/patch/put requests) that the team needs.

### CODE REVIEW

**Comment by mballeza** [comment]
Here are some notes I took while looking at the admin page view. I'm not sure how much needs to be done since your goal is to merge some important changes from other files.
--- As Super Admin ---
-Clickable space extends beyond the checkboxes
-Remove clickable rows

Users Table
-Should the disabled prop look for role === "super_admin" instead of "admin"?

Organizations Table
-Blocked column check marks: "toggleOrganizationVisibility is not a function"
-Add Admin Button -> Add Button: "cannot read property 'id' of undefined


--- As Admin ---
Organizations Table
-Add Admin: "404 Not Found" page from dialog

**Comment by iandusenbury** [comment]
@mballeza 
1. Will fix in a future commit, just want to get this out
2. the prop is `role !== admin` so if the user isn't an admin then the button will be disabled because there isn't anything to revoke.
3. That will work once the endpoints/payloads are added
4. The 404 is a placeholder because I haven't added the other functionality yet.

**Review by halalak — COMMENTED** [review]
can you please elaborate on how the default value 'current' is used, just curious about how that works

**Comment by iandusenbury** [comment]
@halalak `fetchProfessional()` will use 'current', getting the current user (for the logged in user). `fetchProfessional(1)` will fetch the professional with id of 1. You could also call `fetchProfessional('random')` if you wanted to.

File: client/src/components/edit/AddExperience.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 import { StyledTextField } from '../../widgets/StyledTextField'
 import { AddButton } from '../../widgets/AddButton'
 import { StyledSelectField } from '../../widgets/StyledSelectField'
-import { DatePicker, MenuItem } from 'material-ui'
 import { style } from '../../widgets/styles'
 import './edit.css'
 
-export const AddExperience = props => {
+const AddExperience = props => {
   const { type } = props
```
Comment (halalak) [inline client/src/components/edit/AddExperience.js:12]: Is there a reason why this syntax is preferred over the previous way the components were written?

Comment (iandusenbury) [reply client/src/components/edit/AddExperience.js:12]: linter was complaining about the other way. You would use the previous syntax if you were exporting multiple components from the file.

**Review by halalak — APPROVED** [review]
Everything looks good to me, I think it's ready to merge (once it's rebased)!

**Review by mballeza — APPROVED** [review]

**Pull request merged**